hera_sim/simulate.py

```diff
@@ -904,7 +930,7 @@ def _update_history(self, model, **kwargs):
         msg = "hera_sim v{version}: Added {component} using kwargs:\n"
         if defaults._override_defaults:
             kwargs["defaults"] = defaults._config_name
-        for param, value in kwargs.items():
+        for param, value in defaults._unpack_dict(kwargs).items():
             msg += "{param} = {value}\n".format(param=param, value=value)
```

Comment:
can use f-string formatting

Reply:
Yep. This is *old* code, and I didn't know about f-strings at the time. There are a few other instances of this around the code... I was thinking of getting all of these stylistic issues resolved in one fell swoop in a `code_beautification` PR.

Reply:
Updated this to use f-strings here, but there are plenty of other places where this needs to be done... hence the idea of making a `code_beautification` PR.